### ISSUE TITLE
Firebase 11.0.0 support

### ISSUE THREAD

**Issue by alpha-moisol** [customerio/customerio-ios#792]
**Is your feature request related to a problem? Please describe.**
Cannot use latest version of CustomerIO with Firebase >= 11.0.0

**Describe the solution you'd like**
Update CustomerIO library to able to work with Firebase >= 11.0.0

**Comment by mrehan27** [comment]
Hi @alpha-moisol. Thanks for reaching out and sharing this with us. We have noted it down and will be prioritizing and working on it soon. We'll let you know once we roll out updates for this. Appreciate your patience.

**Comment by vendoshi** [comment]
I am facing the same issue while installing customer io sdk

```
[!] CocoaPods could not find compatible versions for pod "FirebaseMessaging":
  In Podfile:
    CustomerIO/MessagingPushFCM (~> 2.14.1) was resolved to 2.14.1, which depends on
      CustomerIOMessagingPushFCM (= 2.14.1) was resolved to 2.14.1, which depends on
        FirebaseMessaging (< 11, >= 8.7.0)

    firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.0.0, which depends on
      Firebase/Messaging (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseMessaging (~> 11.0.0)
```

Is there any temporary solution available until you support 11.0.0? Other than downgrading my firebase sdk?

**Comment by Shahroz16** [comment]
@vendoshi, currently, there isn't a way other than downgrading, but we do have plans to explore this in the recent future. We need to ensure that the upgrade does not disrupt any existing functionality.

**Comment by lukemmtt** [comment]
+1

**Comment by alpha-moisol** [comment]
Hello, it's been over a month since this issue was created. Is there any progress?

**Comment by BenjiDosh** [comment]
Hey @Ahmed-CIO , do you have any indication one when you think this will be fixed?

**Comment by Shahroz16** [comment]
Thank you for the patience here, everyone, I am glad to let you know, that we will be looking into it as a priority and plan to roll out a change soon after we have tested and verified that increasing the version doesn't break compatibility to any other feature. We will let everyone know when the release is out, appreciate the patience here.

**Comment by bfish1996** [comment]
Hi, @Shahroz16 , do you have an estimated timeline on this?

**Comment by alexandersandberg** [comment]
They seem to be working on it in https://github.com/customerio/customerio-ios/pull/817, and it looks like it could land within a few days(?).

**Comment by ami-aman** [comment]
We've released iOS SDK v[2.14.2](https://github.com/customerio/customerio-ios/releases/tag/2.14.2) with Firebase 11 support. Let us know if you still face any issues while upgrading the versions.
